server_admin/topics/clients/con-advanced-settings.adoc

```diff
@@ -1,72 +1,84 @@
 [id="con-advanced-settings_{context}"]
 ==== Advanced Settings
-There are 
+Advanced settings also available for OIDC Clients.
+
 [[_mtls-client-certificate-bound-tokens]]
 *OAuth 2.0 Mutual TLS Certificate Bound Access Tokens Enabled*
 
-Mutual TLS binds an access token and a refresh token with a client certificate exchanged during TLS handshake. This prevents an attacker who finds a way to steal these tokens from exercising the tokens. This type of token is called a holder-of-key token. Unlike bearer tokens, the recipient of a holder-of-key token can verify whether the sender of the token is legitimate.
-
-If the following conditions are satisfied on a token request, {project_name} will bind an access token and a refresh token with a client certificate and issue them as holder-of-key tokens. If all conditions are not met, {project_name} rejects the token request.
+NOTE: To enable mutual TLS in {project_name}, see <<_enable-mtls-wildfly, Enable mutual SSL in WildFly>>.
```

Comment:
Can NOTE work this way?  I thought it had to be
[NOTE]
====
blah, blah, blah
====